inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java

```diff
@@ -311,6 +311,14 @@ public void empty_WithNonemptyKnowledgeBase_ShouldReturnFalse() {
             .isFalse();
     }
 
+    @Test
+    public void nonempty_WithEmptyKnowledgeBase_ShouldReturnTrue() {
+        sut.registerKnowledgeBase(kb, sut.getNativeConfig());
+        sut.defineBaseProperties(kb,false);
+        List<KBHandle> listProperties = sut.listProperties(kb, true);
+        assertEquals(listProperties.size(),3);
```

Comment:
Please also assert that the list contains the IRIs you expect, not only the size.